### PR TITLE
Added discord chat to contact

### LINKED ISSUE #924: Add Discord chat to Documentation
- Mention we now sit in Discord in docs

### PR DESCRIPTION
**Pull request by mohitbalwani**
Removed IRC client and added discord chat in contact section.

Fixes #924

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ The bandersnatch project strives to:
 If you have questions or comments, please submit a bug report to
 https://github.com/pypa/bandersnatch/issues/new
 
-- IRC: #bandersnatch on *Freenode* (You can use [webchat](https://webchat.freenode.net/?channels=%23bandersnatch) if you don't have an IRC client)
+- DISCORD: #bandersnatch now sit in *Discord* (You can use [webchat](https://webchat.freenode.net/?channels=%23bandersnatch) if you don't have Discord account)
```
Comment (cooperlees) [inline README.md:190]:
```suggestion
- Discord: #bandersnatch now sit in the *PyPA Discord* server. To join visit https://discord.com/invite/pypa
```

**Review by cooperlees — CHANGES_REQUESTED** [review]
Thanks! Looks good. Lets just add the link I should have put in the issue.

**Review by cooperlees — APPROVED** [review]
LGTM. Many thanks for your first contribution to bandersnatch!

**Comment by mohitbalwani** [comment]
Thanks for helping me out in this issue! This is my first open source
contribution.

**Pull request merged**